### DISCUSSION TITLE
Possible to run on Pi?

### DISCUSSION THREAD

**Discussion by RichyHBM** [PartialVolume/shredos.x86_64#170] [Ideas]
Hi, I've been looking for uses for some old pis and thought of turning one into a shredder device, I'm not sure how much CPU is needed to run nwipe if it would be feasible, but if so would it be possible to provide an arm64 variant of the images?

**Comment by PartialVolume** [comment]
CPU and memory wise, a old pi would work ok, however you really need USB V3, If I remember correctly anything less than an RPi4 would have USB v2 so that may cause the wipe to be much slower especially if the drive is capable of upwards of 60MByte/s. An RPi4 would be an ideal device because you can also boot from a USB stick unlike the earlier pi's so I believe.

I could build an arm variant but at the moment I just don't have the time to spend on it. I'm hoping somebody else will come along and take up that challenge and post a detailed procedure for taking the existing ShredOS, listing the changes required will full instructions for burning to SDCARD or in the case of RPi4 modifying the boot firmware to boot from USB.
